api/endpoints/zoom.js

```diff
@@ -62,7 +62,7 @@ async function handleEvent(req, res, meeting) {
     case 'meeting.ended':
       await Prisma.create('customLogs', { text: 'zoom_end_meeting_webhook', zoomCallId: meeting.zoomID || "undefined" })
       console.log('Attempting to close call w/ ID of', )
-      await closeZoomCall(meeting.zoomID, false)
+      await closeZoomCall(meeting.zoomID, true)
```

Comment:
Are you sure about this?

Reply:
once we get a meeting.ended event we can be sure the meeting has to be closed right?

Reply:
Sgtm